staging/src/k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1/constants.go

```diff
@@ -30,6 +30,15 @@ const (
 	DevicePluginPath = "/var/lib/kubelet/device-plugins/"
 	// KubeletSocket is the path of the Kubelet registry socket
 	KubeletSocket = DevicePluginPath + "kubelet.sock"
+
+	// Avoid failed to run Kubelet: bad socketPath,
+	// must be an absolute path: /var/lib/kubelet/device-plugins/kubelet.sock
+	// https://github.com/kubernetes/kubernetes/issues/93262
+	// https://github.com/kubernetes/kubernetes/pull/93285#discussion_r458140701
+	DevicePluginPathWindows = "c:\\var\\lib\\kubelet\\device-plugins\\"
```

Comment:
will it always be on c:\ drive?

Reply:
The clean solution seems to be https://github.com/kubernetes/kubernetes/pull/91118.
But because the path was hardwired on Unix so far, I think doing the same for Windows should not block this PR.

Reply:
It's seem they use systemDrive
```
$global:KubernetesPath = "$env:SystemDrive\k"
```
https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/scripts/PrepareNode.ps1

Reply:
should you also use $env:SystemDrive in your PR instead of C:\ then?

Reply:
> should you also use $env:SystemDrive in your PR instead of C:\ then?

But then applying #91118 first seems cleaner to me.

Reply:
> should you also use $env:SystemDrive in your PR instead of C:\ then?

yes, I just add it

https://github.com/kubernetes/kubernetes/pull/93285/commits/e03f8287c942709cb1ddeaf17f0ea6126caf2095#diff-b54ee8195220ea88e59b6eb423e83dd1R131

Reply:
> 
> 
> > should you also use $env:SystemDrive in your PR instead of C:\ then?
> 
> yes, I just add it
> 
> [e03f828#diff-b54ee8195220ea88e59b6eb423e83dd1R131](https://github.com/kubernetes/kubernetes/commit/e03f8287c942709cb1ddeaf17f0ea6126caf2095#diff-b54ee8195220ea88e59b6eb423e83dd1R131)

Yes, that's nice!
Then #91118 can be done separately.